common/static/common/templates/discussion/discussion-home.underscore

```diff
@@ -8,7 +8,10 @@
 
   <% if (window.ENABLE_DISCUSSION_HOME_PANEL) { %>
     <span class="label label-settings">
-      <%- interpolate(gettext("How to use %(platform_name)s discussions"), {platform_name: window.PLATFORM_NAME}, true) %>
+      <%- interpolate(
+            gettext("How to use %(platform_name)s discussions"),
+            {platform_name: window.PLATFORM_NAME}, true
```

Comment:
@singingwolfboy gettext doesn't work in the same line as interpolate in underscore templates. Please be sure that you're properly testing your features for i18n.